platforms/hyperledger-fabric/charts/invoke_chaincode/templates/invoke_chaincode.yaml

```diff
@@ -144,7 +144,7 @@ spec:
             echo $ENDORSING_ORG_NAMES
             for ns in $ENDORSING_ORG_NAMES
             do
-              SECRETPREFIX=secret/crypto/peerOrganizations/${CREATOR_NS}/endorsers/${ns}/users/admin
+              SECRETPREFIX=secretsv2/data/crypto/peerOrganizations/${CREATOR_NS}/endorsers/${ns}/users/admin
```

Comment:
I think fetching the `secret or secretv2` prefix from network.yaml is better.
Lets have a discussion around this today.